include/hlsl++/common.h

```diff
@@ -248,7 +248,9 @@ namespace hlslpp
 #endif
 
 	// Reference http://www.liranuna.com/sse-intrinsics-optimizations-in-popular-compilers/
+#if !defined(_hlslpp_sign_ps)
```

Comment:
The define should probably go above the comment to indicate that the link refers to the line below